src/app/pages/admin-page/admin-page-routing.module.ts

```diff
@@ -17,33 +18,49 @@ const routes: Routes = [
         component: VendorTableComponent
       },
       {
-        path: 'vendors/:id',
+        path: 'vendors/form',
```

Comment:
don't you want to make them 'children'?

Reply:
I've tried, but got some unexpected behavior. Let's left this to the next PR. It's not the end of this task)

Reply:
ok, np)